CHANGELOG.md

```diff
@@ -5,6 +5,8 @@
 * Adds new processor types to `Minfraud::Components::Payment`: `:cashfree`,
   `:first_atlantic_commerce`, `:komoju`, `:paytm`, `:razorpay`, and
   `:systempay`.
+* Adds support for the new Factors outputs `/subscores/device`,
+  `/subscores/email_local_part` and `/subscores/shipping_address`
```

Comment:
Should we have a period? It might also be good to give a brief description of them here.